test/Unit/DependencyManagement/DependencyManagementTests.cs

```diff
@@ -449,6 +449,62 @@ public void FunctionAppExecutionShouldContinueIfPreviousDependenciesExist()
                 TestCaseCleanup();
             }
         }
+
+        [Theory]
+        [InlineData("CONTAINER_NAME", "MyContainerName", true)]
+        [InlineData("WEBSITE_INSTANCE_ID", "MyInstanceId", true)]
+        [InlineData(null, null, false)]
+        public void ValidateManagedDependenciesPath(string name, string value, bool setEnvironmentVariable)
+        {
+            const string HomeDriveName = "HOME";
+
+            bool deleteVariable = false;
+            string expectedPath = null;
+
+            if (setEnvironmentVariable)
+            {
+                Environment.SetEnvironmentVariable(name, value);
+                Environment.SetEnvironmentVariable(HomeDriveName, "home");
+
+                deleteVariable = true;
+
+                const string DataFolderName = "data";
+                const string ManagedDependenciesFolderName = "ManagedDependencies";
+                expectedPath = Path.Combine(HomeDriveName, DataFolderName, ManagedDependenciesFolderName);
+            }
+            else
+            {
+                string appDataFolder = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.DoNotVerify);
+                expectedPath =  Path.Combine(appDataFolder, AzureFunctionsFolderName, "MyFunction", ManagedDependenciesFolderName);
+            }
+
+            var functionAppRoot = Path.Join(Path.GetTempPath(), "MyFunction");
+
+            try
+            {
+                if (!Directory.Exists(functionAppRoot))
```

Comment:
No need for this check: `Directory.CreateDirectory` will not fail if the directory already exists

Reply:
Fixed.